xrpl/asyncio/ledger/main.py

```diff
@@ -54,13 +55,26 @@ async def get_fee(client: Client) -> str:
         client: the network client used to make network calls.
 
     Returns:
-        The minimum fee for transactions.
+        The minimum fee for transactions, in drops.
```

Comment:
```suggestion
        The load-scaled transaction cost, in drops.
```

Reply:
Hm, would this be a breaking change? Should we just change how autofill does it?

Reply:
I guess you could view it that way, but I think it's more like a documentation fix.

Reply:
I was referring to the change itself - this seemed like a good illustration of my concern.